### DISCUSSION TITLE
Questions about Cora dataset

### DISCUSSION THREAD

**Discussion by scottshufe** [pyg-team/pytorch_geometric#5546] [Q&A] [answered]
1. First, when I use `dataset = Planetoid(path, 'Cora')[0]` to load Cora dataset, and 
```
from torch_geometric.utils import to_networkx

G = to_networkx(data)
G = nx.to_undirected(G)
G = nx.Graph(G)
nx.info(G)
```
I got `'Graph with 2708 nodes and 5278 edges'`. 5278 = 10556 / 2, there is no problem.

2. Second, when I use the RandomLinkSplit transformation T.RandomLinkSplit(num_val=0.05, num_test=0.1, is_undirected=True, add_negative_train_samples=False) to load Cora `dataset = Planetoid(path, 'Cora', transform)[0]` and I get the train_data (8976 edges), val_data, test_data. Then, 
```
from torch_geometric.utils import to_networkx

G = to_networkx(train_data)
G = nx.to_undirected(G)
G = nx.Graph(G)
nx.info(G)
```
I got `'Graph with 2708 nodes and 3513 edges'`

Then, my questions is why the edge num of  the obtained graph (3513) does not equal to the half of train_data edge num (8976/2=4488)

**Comment by EdisonLeeeee** [comment, marked as answer]
Which PyG version are you using?  This bug has been resolved for PyG>=2.0.4

**Reply by scottshufe** [reply]
My PyG version is 2.0.2. I'll check it by newer PyG version that >= 2.0.4. Thanks.

**Reply by scottshufe** [reply]
I have checked this bug for PyG == 2.1.0 and it has been resolved. Thanks for your help.
